ibm/service/cis/resource_ibm_cis_mtls.go

```diff
@@ -0,0 +1,217 @@
+// Copyright IBM Corp. 2017, 2022 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package cis
+
+import (
+	"fmt"
+
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/conns"
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/flex"
+	"github.com/IBM/go-sdk-core/v5/core"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+const (
+	cisMtlsID            = "mtls_id"
+	cisMtlsCert          = "cert_mtls"
+	cisMtlsCertName      = "cert_name"
+	cisMtlsHostName      = "host_name"
+	cisMtlsCertCreatedAt = "created_at"
+	cisMtlsCertUpdatedAt = "updated_at"
+	cisMtlsCertExpireOn  = "expires_on"
+)
+
+func ResourceIBMCISMtls() *schema.Resource {
+	return &schema.Resource{
+		Create:   ResourceIBMCISMtlsCreate,
+		Read:     ResourceIBMCISMtlsRead,
```

Comment:
Don't expose methods make "R" to small"r"

Reply:
Thanks for pointing this out.